doc/integrator/headers.rst

```diff
@@ -0,0 +1,26 @@
+.. _integrator_headers:
+
+Headers
+=======
+
+We can set with some configuration some HTTP headers on defferent path.
+We do that with the ``global _headers`` from the configuration with the following syntax:
+
+.. code:: yaml
+
+    vars:
+      global_headers:
+        - pattern: <regex>
+          headers:
+            <header>: <value>
+
+If a path match more than one pattern, all the headers will be applyed, if one header applyed tweice
+the last one win.
+
+For the ``Content-Security-Policy`` we add some vars to be customisable.
```

Comment:
Not sure I understand... is it:

For the ``Content-Security-Policy`` headers in particular, c2cgeoportal includes specific variables in its standard template, to facilitate the customization of these values.

?